### ISSUE TITLE
tmux-2.2 utf-8 character problem on SmartOS Zone 

### ISSUE THREAD

**Issue by drscream** [TritonDataCenter/pkgsrc-legacy#397]
I've installed a 2016Q2 zone (base-64-16.2.0) and installed the current `tmux`-package in version 2.2. I've updated and modified the `locale` to set by default `en_US.UTF-8`.

For me it's not possible to insert or show UTF-8 characters in `tmux`.

```
$ pkgin in tmux
$ export LC_ALL=en_US.UTF-8
$ tmux
```

To be sure i've exported the `LC_ALL` variable again and also started a new `bash`:

```
tmux> export LC_ALL=en_US.UTF-8
tmux> bash
```

Inserting characters like `𝄞` or `»` isn't possible and are not shown in `tmux`. This only happen on SmartOS and only with the new version 2.2.

**Comment by mamash** [comment]
I can verify that and seems to be same problem as described in this upstream bug:

https://github.com/tmux/tmux/issues/463

**Comment by mamash** [comment]
As the upstream bug suggests, tmux built from master doesn't exhibit this bug, so it will be fixed in the next release. However, I couldn't identify any particular commit that we could backport for a fix.

This problem will be fixed in the next pkgsrc branch after tmux 2.3 ships, and we can backport the fix to the 2015Q4 LTS then.

**Comment by mamash** [comment]
tmux-2.3 is available in pkgsrc trunk now, and I just backported the update the 2015Q4 LTS release.
